pom.xml

```diff
@@ -90,7 +90,10 @@
 			<version>${liquibase.version}</version>
 		</dependency>
 
-
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
```

Comment:
It's better to use spring-boot-starter-validation which provides hibernate-validator also